src/attack.c

```diff
@@ -519,13 +522,21 @@ attack_host (struct scan_globals *globals, struct in6_addr *ip, GSList *vhosts,
                   kb_item_push_str (main_kb, "internal/results", buffer);
                   goto host_died;
                 }
+              else if (e == ERR_NO_FREE_SLOT)
+                {
+                  g_debug (
+                    "fork() failed for %s. No free slot to run a plugin.",
```

Comment:
`fork()` was never tried here. ERR_NO_FREE_SLOT is returned long before `create_process()` is called.
Maybe better debug message would be  `Launch failed for %s. No free slot to run a plugin."`